### PR TITLE
Correct typos

### PR DESCRIPTION
**Pull request by bf4**
Mostly fixed by running

```sh
misspell  -w -error -source=text .
```

First noticed while going through the guides

FWIW, specifically related to

- https://github.com/ember-learn/guides-source/pull/1247
- https://github.com/ember-learn/guides-source/pull/1248
- https://github.com/ember-learn/guides-source/pull/1249
- https://github.com/ember-learn/guides-source/pull/1250

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -543,7 +543,7 @@ Fixes a few issues:
 - [#15849](https://github.com/emberjs/ember.js/pull/15849) [BUGFIX] Fix issue when observing a computed property that is clobbered during creation.
 - [#15797](https://github.com/emberjs/ember.js/pull/15797) [BUGFIX] Fix issues with using partials nested within other partials.
 - [#15808](https://github.com/emberjs/ember.js/pull/15808) [BUGFIX] Fix a memory leak in certain testing scenarios.
-- [#15746](https://github.com/emberjs/ember.js/pull/15746) [BUGFIX] Fix computed sort regression when array property is initally `null`.
+- [#15746](https://github.com/emberjs/ember.js/pull/15746) [BUGFIX] Fix computed sort regression when array property is initially `null`.
```
Comment (locks) [inline CHANGELOG.md]: Can you revert the changes to CHANGELOG.md please?

**Review by locks — CHANGES_REQUESTED** [review]
Thanks for the help! Left a small request inline.

File: CHANGELOG.md (hunk 1 above)
Comment (bf4) [reply CHANGELOG.md]: sure. done

**Review by locks — APPROVED** [review]
Thanks, looking good!

**Review by rwjblue — APPROVED** [review]
